docs/source/pipeline.rst

```diff
@@ -0,0 +1,291 @@
+=========
+Pipelines
+=========
+
+Vertica Pipelines is an open source platform for managing data
+scientists’ machine learning pipelines. They are built on a
+human-readable data format: **YAML**.
+
+==============================
+Setting Up Your First Pipeline
+==============================
+
+Requirements
+~~~~~~~~~~~~
+| To begin, you must:
+| - Have access to a machine that has Vertica installed
+| - Install Python on your machine
+| - Install Verticapy
+
+Create Your First YAML files
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+   | **THE BASIC RULES OF ANY YAML FILE:**
```

Comment:
This creates an unnecessary indentation. Was it intended? If no then you can remove the space before "|". In fact, it is even better to just remove "|" altogether.  "|" is used where you want to force it to use separate line.

![image](https://github.com/vertica/VerticaPy/assets/46414488/b21e014d-5a5b-448e-bc24-9362ce9b0fcb)